guides/production.md

```diff
@@ -82,10 +82,138 @@ All data transmitted to and from your Moovweb XDN project must be secured with T
 
 The XDN provides a wildcard TLS certificate that covers the auto-generated domains that it assigns to your site (e.g {team}-{site}-{branch}-{version}.moovweb.io). You need to provide your own certificate for your site's custom domains.
 
-### Obtaining a certificate
-
 _Note: If you already have an existing certificate, you can use it by skipping ahead to [Uploading your certificate](#section_uploading_your_certificate). Many customers who have existing certificates still choose to obtain a new one when adopting the XDN so as not to reuse the same private key with more than one vendor/system_
 
+### Obtain a certificate automatically from the XDN Console
+
+The XDN console can use [**Let's Encrypt**](https://letsencrypt.org/) to generate a SSL Certificate on your behalf and deploy it on your XDN website. This certificate is free, it is valid for 3 months, and will be automatically renewed as long as the technical requirements, shown below, remains met.
+
```

Comment:
After further thought @Th0masL I think we should essentially add a TOC or Tl;dr to the top of this section covering the required steps in 4 lines so people don't need to read through the whole doc to understand what they need to do.

Then link to an anchor for each of the 4 sections below it.

Reply:
I'm not exactly sure what you mean by that, since people actually do have to read the whole section to understand what they have to do, plus the doc is not that long, so that shouldn't be much of a problem.

But I'm happy to apply changes if we decide they are needed. Let's see on Monday :)